crons.md

```diff
@@ -30,6 +30,10 @@ For example, let's say we have a cron that runs once a minute (a `cron_frequency
 - If the amount of data processed could increase and cause this cron to take 90 seconds to finish, any `cron_max_run_time` less than 90 seconds would be unsafe.  Even if concurrent crons are not problematic for the module itself, this could cause the number active cron processes to pile up over time and crash the server.
 - If the amount of data processed could increase and cause this cron to occasionally take hours to finish, it may be prudent to set much larger `cron_max_run_time` to be safe (perhaps 24 hours, or `86400` seconds).
 
+#### Crons That Run Long And Often
+
+Sometimes modules have large backlogs of work that they are processing constantly (e.g. API Sync, Flight Tracker).  On most REDCap instances it is considered inappropriate for a module to be designed and/or configured such that it runs constantly, consumes an outsize portion of system resources, and/or affects system performance in general.  Each module cron is intentionally limited to running once at a time to limit effects on the system, and prevent any thread safety issues from code that is not safe to run concurrently.  However, there are cases where more background work is desired that a single constantly running cron can handle.  In these rare cases, it is recommended to create additional crons that perform the same task (after checking the code to ensure thread safety).  For example, these crons could be named `intensive_background_tasks`, `intensive_background_tasks2`, etc.).  If this module will be distributed to other systems, is recommended to only run one intensive background task queue by default, perhaps requiring an admin to set a system setting on your module before additional similar crons will consume additional resources.  We considered adding support for `redcap_crons.cron_instances_max`, but decided against that as it would be harder to control on a per system basis, and may be more likely to encourage misuse of system resources.
```

Comment:
```suggestion
Sometimes modules have large backlogs of work that they are processing constantly (e.g. API Sync, Flight Tracker).  On most REDCap instances it is considered inappropriate for a module to be designed and/or configured such that it runs constantly, consumes an outsize portion of system resources, and/or affects system performance in general.  Each module cron is intentionally limited to running once at a time to limit effects on the system, and prevent any thread safety issues from code that is not safe to run concurrently.  However, there are cases where more background work is desired that a single constantly running cron can handle.  In these rare cases, it is recommended to create additional crons that perform the same task (after checking the code to ensure thread safety).  For example, these crons could be named `intensive_background_tasks`, `intensive_background_tasks2`, etc.).  If this module will be distributed to other systems, it is recommended to only run one intensive background task queue by default, perhaps requiring an admin to set a system setting on your module before additional similar crons will consume additional resources.  We considered adding support for `redcap_crons.cron_instances_max`, but decided against that as it would be harder to control on a per system basis, and may be more likely to encourage misuse of system resources.
```

Minor grammar correction